pymc3/sampling.py

```diff
@@ -81,7 +81,7 @@ def assign_step_methods(model, step=None, methods=(NUTS, HamiltonianMC, Metropol
     return steps
 
 
-def sample(draws, step=None, init='advi', n_init=200000, start=None,
+def sample(draws, step=None, n_advi=200000, n_nuts=2000, start=None,
```

Comment:
Good point. I'm starting to think about this as init-programs. This new one uses ADVI+NUTS. Perhaps just an init_kwargs dict that can get passed is most general?